SetupDataPkg/Tools/KnobService.py

```diff
@@ -300,6 +318,7 @@ def generate_public_header(schema, header_path, efi_type=False):
             out.write("#include <stdint.h>" + get_line_ending(efi_type))
             out.write("#include <stddef.h>" + get_line_ending(efi_type))
             out.write("#include <stdbool.h>" + get_line_ending(efi_type))
+        out.write(get_schema_header_references(schema, efi_type))

```

Comment:
Maybe a nit, but it feels like this should go after the comment indicating that this is auto-generated code / the source for the generated content.

Reply:
Agreed, will change